mlos_bench/mlos_bench/services/remote/azure/README.md

```diff
@@ -0,0 +1,37 @@
+# Azure Services
+
+## Authentication
+
+This service allows us to generate tokens for authenticating with other Azure services.
+It supports authenticating as the following identities:
+
+### User
+
+As a user, the only requirement is that a user is logged in via the Azure CLI before `mlos_bench` is started i.e. `az login` or `az login --device-login`.
+This method is simpler to get started, but the drawback is that organization policies might limit the period in which we can generate access tokens under a user logged in via the CLI.
+This would lead to early termination of an experiment once tokens cannot be generated anymore without another interactive login by the user.
+
+### Service Principal (SP)
+
+Similar to above, it also requires a user to be logged in first the Azure CLI to kick off the flow for the SP.
+However, this method addresses the drawback above to some extent as the login for a SP can be set to expire much later i.e. when the associated certificate expires in a year.
```

Comment:
```suggestion
However, this method addresses the drawback above to some extent as the since we can configure the system to automatically reauthenticate as the SP on demand and restrict its access to fewer resources to mitigate security concerns so that experiments can run as long as necessary.
```